src/PageValidator.php

```diff
@@ -30,7 +30,7 @@ class PageValidator extends AbstractValidator
         ],
         'content' => [
             'required',
-            'max:65535',
+            'max:16777215',
```

Comment:
I'd even remove this line entirely. No one will write ever write more than 16 million characters.

Reply:
I don't think validation can hurt